### PR TITLE
Restructure

### PR DESCRIPTION
**Pull request by DrunkenImpGameStudio**
Restructure

Removed ITerminal (#92   ##Completed)

Renamed Terminal -> TerminalOptions

### CODE REVIEW

**Comment by DrunkenImpGameStudio** [comment]
Opps, sorry cat walked across keyboard. lol.

File: src/WarriorRogueMage/CharacterCreation/SetAttributesState.cs [hunk 1]
```diff
@@ -157,22 +157,22 @@ private void SetAttributeTarget(ref int targetPoints, int newValue)
...
-            sb.AppendAnsiLine($"Mage    : {magePoints}" + AnsiSequences.NewLine);
-            sb.AppendAnsiLine();
-            sb.AppendAnsiLine($"You have {MaxPoints - SpentPoints} character points left.");
-            sb.AppendAnsiLine();
-            sb.AppendAnsiLine("<%yellow%>====================================================================");
-            sb.AppendAnsiLine("To add points to an attribute, use the + operator. Example: warrior +6");
-            sb.AppendAnsiLine("To subtract points from an attribute, use the - operator. Example: warrior -6");
-            sb.AppendAnsiLine("When you are done distributing the character points, type done.");
-            sb.AppendAnsiLine("====================================================================<%n%>");
+            sb.AppendLine();
```
Comment (DavidRieman) [inline src/WarriorRogueMage/CharacterCreation/SetAttributesState.cs:160]: Odd regression showing up here.
Also with so many commits showing up in the commit list on each PR, I wonder if your fork needs a fresh merge from the base repo master?

**Review by DavidRieman — CHANGES_REQUESTED** [review]
You've got deletions of the base /systemdata/ files that need to come back, and another issue below.  Haven't reviewed thoroughly yet but I'll be back.

**Comment by DrunkenImpGameStudio** [comment]
Yeah, i had to fetch because of the changes before (this was the one I started prior about needing a restructure then came back to)

I actually wanted to ask if you would be down for coded slash/credits. Ive already created the code. It allows you to add/remove credits in game, and to store the names to DB. Then builds the credit block on the fly. Same with the splash, it also allows you to specify a specific splash (for like holidays or special events)

**Review by DavidRieman — COMMENTED** [review]
Yeah, pulling latest / cleaning up the commits list will help expose whether the SetAttributesState is a real regression this PR would push through, or just a GitHub hiccup (I've seen this sort of thing rarely in the past).

For splash screens and credits:  I've opened a [discussion thread](https://github.com/DavidRieman/WheelMUD/discussions/104) so we don't lose track of it here when the PR merges.

File: src/Core/Renderer/RendererDefinitions.cs [hunk 2]
```diff
@@ -5,11 +5,11 @@
...
 //-----------------------------------------------------------------------------
 
-namespace WheelMUD.Core
-{
-    using System.Collections.Generic;
-    using WheelMUD.Interfaces;
+using System.Collections.Generic;
+using WheelMUD.Server;
 
+namespace WheelMUD.Core.Renderer
```
Comment (DavidRieman) [inline src/Core/Renderer/RendererDefinitions.cs:11]: I'm not really sold on these or Behaviors becoming their own namespace again.  I suspect it'll feel better if Interfaces also are not isolated from their more general namespace.  We used to have way too many interfaces before, and it was harder to discover things through IntelliSense and the like, so we actively removed a bunch of them.  As evidence of this, the current code's commented-out CreateCurrency command shows "using WheelMUD.Universe.Behaviors;" which you're resurrecting with this PR.

This is not a blocking concern for me, but something we can discuss.  :)

**Pull request closed without merging**